projekt_do_pracy_na_labach/fiszki_app/Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN flutter config --enable-web
 RUN mkdir /app/
 COPY . /app/
 WORKDIR /app/
-RUN flutter build web --dart-define BACKEND_URL=http://backend-service
+RUN flutter build web --dart-define BACKEND_URL=http://localhost:8000/
```

Comment:
Tu jest wina w sumie twórcy instrukcji (moja), jak usuniesz `http://` to będzie już działać.  
Ale gratuluję bo chyba jeszcze nikt na to nie wpadł by wrzucić localhost'a (co w sumie w przypadku tych labów jest rozwiązaniem). Jakiegoś plusa dopiszę